test/spoom/cli/commands/config_test.rb

```diff
@@ -98,6 +110,84 @@ def test_display_config_with_ignored_files
             Paths ignored:
              * lib/main
              * test
+
+            Allowed extensions:
+             * .rb (default)
+             * .rbi (default)
+          MSG
+        end
+
```

Comment:
Does it make sense to add tests of the following:

- `--allowed-extension` with extensions after newline
- accidental misuse of lines matching `^--allowed-extension$`, `^--$`, and `^--ignore$` (would block proper processing of next line)
- combinations of `^--allowed-extension$`, `^--$`, and `^--ignore$` in the same config file?